cpp/ql/lib/semmle/code/cpp/ir/implementation/EdgeKind.qll

```diff
@@ -123,9 +145,22 @@ module EdgeKind {
   FalseEdge falseEdge() { result = TFalseEdge() }
 
   /**
-   * Gets the single instance of the `ExceptionEdge` class.
+   * Gets an instance of the `CppExceptionEdge` class.
+   */
+  CppExceptionEdge cppExceptionEdge() { result = TCppExceptionEdge() }
+
+  /**
+   * Gets an instance of the `SehExceptionEdge` class.
    */
-  ExceptionEdge exceptionEdge() { result = TExceptionEdge() }
+  SehExceptionEdge sehExceptionEdge() { result = TSehExceptionEdge() }
+
+  /**
+   * Gets an instance of the `ExceptionEdge` class.
+   */
+  ExceptionEdge exceptionEdge() {
+    result = cppExceptionEdge() or
+    result = sehExceptionEdge()
+  }
```

Comment:
I would not introduce this predicate, unless you need it somewhere later. I don't think it's being used right now, correct?

Reply:
Only introduced to keep parity with what we had. I'll delete it.

Reply:
Done